### PR TITLE
out_s3: add docs for canned ACL and content options

### PR DESCRIPTION
**Pull request by zhonghui12**
Signed-off-by: Zhonghui Hu <zh0512xx@gmail.com>

### CODE REVIEW

**Comment by zhonghui12** [comment]
@PettitWesley  here is the PR for branch `1.6`

File: pipeline/outputs/s3.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ Records are stored in files in S3 as newline delimited JSON.
 | role\_arn | ARN of an IAM role to assume \(ex. for cross account access\). | None |
 | endpoint | Custom endpoint for the S3 API. | None |
 | sts\_endpoint | Custom endpoint for the STS API. | None |
+| canned\_acl | Predefined Canned ACL policy for S3 objects. | None |
```
Comment (PettitWesley) [inline pipeline/outputs/s3.md:28]: For this one, lets add a link to the canned ACL docs

Comment (zhonghui12) [reply pipeline/outputs/s3.md]: sure, have added the link for canned ACL.

**Review by PettitWesley — APPROVED** [review]

**Comment by PettitWesley** [comment]
Since 1.6 is not being developed anymore this is no longer needed

**Comment by zhonghui12** [comment]
This is closed since 1.6 docs is not updated

**Pull request closed without merging**